Tools/CISupport/ews-build/factories.py

```diff
@@ -375,3 +375,11 @@ def __init__(self, platform, **kwargs):
         self.addStep(UpdatePullRequest())
         self.addStep(PushCommitToWebKitRepo())
         self.addStep(SetBuildSummary())
+
+
+class SafeMergeQueueFactory(factory.BuildFactory):
+    def __init__(self, platform, configuration=None, architectures=None, additionalArguments=None, **kwargs):
+        super(SafeMergeQueueFactory, self).__init__()
+
+        self.addStep(RetrievePRDataFromLabel('WebKit/WebKit', label='\"safe-merge-queue\"'))
+        self.addStep(RetrievePRDataFromLabel('apple/WebKit', label='\"safe-merge-queue\"'))
```

Comment:
Can we just iterate through all our remotes here? They should be stored in `GITHUB_PROJECTS`.